storage/memory/storage.go

```diff
@@ -165,6 +154,26 @@ func (s *storage) GetHighestScoredElements(key string, maxElements int) ([]strin
 	return scoredElements, nil
 }
 
+func (s *storage) GetRandomKey() (string, error) {
+	s.Mutex.Lock()
+	defer s.Mutex.Unlock()
+
+	// TODO
```

Comment:
We need to implement the pseudo random key access for the memory storage. Once there is an implementation, it needs to be tested as well.